app/views/users/edit.html.haml

```diff
@@ -30,8 +32,7 @@
 
       %p
         My phone number is
-        = telephone_field_tag "mobile_phone[number]", @mobile_number,
-                              placeholder: "07123 456 789"
+        = telephone_field_tag "mobile_phone[number]", @mobile_number
```

Comment:
Let the validation lib show a placeholder that's appropriate to the selected country.

Reply:
Makes sense.